tests/test_save_load.py

```diff
@@ -408,6 +408,82 @@ def test_save_load_policy(tmp_path, model_class, policy_str):
         os.remove(tmp_path / "actor.pkl")
 
 
+@pytest.mark.parametrize("model_class", [DQN])
```

Comment:
Why do we have this as a parameter when we only set it to one value? Upgrade potential?

Reply:
hmm, I wrote that test when I found the bug reviewing QR-DQN PR on the contrib repo.
So, I think my idea was that QR-DQN will be part of SB3 main repo at some point, hence the parameter ;).